synapse/handlers/room_summary.py

```diff
@@ -981,19 +981,24 @@ async def _build_room_entry(self, room_id: str, for_federation: bool) -> JsonDic
             current_state_ids[(EventTypes.Create, "")]
         )
 
+        aliases = await self._store.get_aliases_for_room(room_id)
+
         entry = {
             "room_id": stats["room_id"],
             "name": stats["name"],
             "topic": stats["topic"],
             "canonical_alias": stats["canonical_alias"],
+            "aliases": aliases,
```

Comment:
It looks like Synapse has stopped returning the `aliases` key as part of the room list in #6970. This was included in the work for #6898, but doesn't seem like it was included in MSC2432?

Reply:
ugh, will unpack this and figure out what needs to be done then. Most likely either a small MSC or acceptance of fate.

Reply:
unpacked as https://github.com/matrix-org/matrix-doc/pull/3624 - will remove the field.

Reply:
Thank you! 👍